CONTRIBUTING.md

```diff
@@ -5,22 +5,27 @@ We welcome your input! 👐
 We want to make it as simple and straightforward as possible to contribute to this project, whether it is a:
 
 - Bug Report
+- Task
 - Discussion
 - Feature Request
 - Creating a Pull Request (PR)
 - Becoming a maintainer
 
 ## Bug Report
 
-We use GitHub issues to track the bugs. Report a bug by using our Bug Report Template in [Issues](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&template=bug_report.md).
+We use GitHub issues to track the bugs. Report a bug by using our Bug Report Template in [Issues](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&template=bug_report.yaml&title=%5BBug%5D%3A+).
+
+## Tasks
+
+We use GitHub issues to track tasks as well. If you are implementing a feature or refactoring a piece of code, you can use the Task Template in [Issues](https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=Task&template=task.yaml&title=%5BTask%5D%3A+).
```

Comment:
Maybe we should mention that this is mainly meant for maintainers of the repo? I guess we don't want external contributors to start submitting task issues.